deploy/helm/kuberhealthy/templates/check-reaper.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if .Values.checkReaper.enabled }}
```

Comment:
I'm hesitant to make check-reaper configurable in helm... Only concern being that if folks are using Kuberhealthy 2.1.0+, checks will not get cleaned up. If we were to make it configurable in Helm, it would be definitely have to be enabled by default and customers should be warned about disabling it